lib/site.jsonnet

```diff
@@ -0,0 +1,132 @@
+// Extract the last octet as an integer.
```

Comment:
All these functions are intricate enough  that maybe they count as code?

And should have unit tests?

https://github.com/yugui/jsonnetunit

Reply:
That framework may be overkill, but a few `asserts` in here would make me feel better.